tests/main.cpp

```diff
@@ -964,3 +964,20 @@ TEST_CASE("getSignaturesForAddress") {
       solana::GetSignatureAddressConfig{2});
   CHECK_EQ(SignaturesForAddress.size(), 2);
 }
+
+TEST_CASE("getTokenLargestAccounts") {
+  const auto connection = solana::rpc::Connection(solana::MAINNET_BETA);
+  const auto TokenLargestAccounts = connection.getTokenLargestAccounts(
+      "1YDQ35V8g68FGvcT85haHwAXv1U7XMzuc4mZeEXfrjE");
+  CHECK_EQ(TokenLargestAccounts.value.size(), 20);
+}
+
+TEST_CASE("getBlocks") {
+  const auto connection = solana::rpc::Connection(solana::MAINNET_BETA);
+  const auto slot = connection.getSlot();
+  uint64_t startslot = slot - 10;
+  uint64_t latestslot = slot;
+  const auto Blocks = connection.getBlocks(startslot, latestslot);
+  CHECK_EQ(Blocks.size(), latestslot - startslot + 1);
```

Comment:
May be some blocks are missed.
I guess you can check block size > 0 and then each block fetch falls between startslot and latestslot.

Reply:
@godmodegalactus  Done the changes. Thanks